atom/browser/api/atom_api_app.cc

```diff
@@ -830,6 +831,26 @@ void App::SetAppPath(const base::FilePath& app_path) {
   app_path_ = app_path;
 }
 
+#if !defined(OS_MACOSX)
+void App::SetAppLogsPath(mate::Arguments* args) {
+  std::string custom_path;
```

Comment:
`base::FilePath` converter should work here.

Reply:
Thank you guys!